### PR TITLE
APPDUX-228: Fixes to incorporate latest PF4 breaking change release

### PR DESCRIPTION
**Pull request by mfrances17**
## Motivation
https://issues.redhat.com/browse/APPDUX-228

## What
Update to the latest Patternfly 4 build and make fixes so the UI behavior and look and feel work as expected.

## Why
To incorporate the latest PF4 components going forward.

## Verification Steps
Open Solution Explorer and traverse the entire UI to verify that no new issues were introduced.

## Checklist:
- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

## Progress
- [x] Finished task

## Additional Notes
You can test effectively by pulling down this branch and running locally, but if you want to test on a server, you can use this docker image of the solution explorer:

docker.io/mfrances17/dev-tutorial-web-app:pf4-latest

**Various screen caps:**
UI-pf4-1

UI-pf4-2

UI-pf4-3

UI-pf4-4

UI-pf4-5

UI-pf4-6

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
+  "version": "2.26.1",
   "private": true,
   "proxy": "http://localhost:5001/",
   "dependencies": {
     "@fortawesome/fontawesome-free": "5.7.2",
-    "@patternfly/patternfly": "2.43.1",
-    "@patternfly/react-core": "3.124.1",
-    "@patternfly/react-icons": "^3.14.23",
+    "@patternfly/patternfly": "4.10.29",
+    "@patternfly/react-core": "4.18.1",
```
Comment (jenny-s51) [inline package.json:9]: can we bump to `4.18.13`?

**Comment by dlabaj** [comment]
Lgtm

File: package.json (hunk 1 above)
Comment (mfrances17) [reply package.json]: I've updated to the latest PF:
patternfly: 4.16.7
react-core: 4.23.1

**Comment by mfrances17** [comment]
@jenny-s51 @dlabaj @christiemolloy @pb82 @tiffanynolan Updated with the latest Patternfly 4 release... can be tested locally or on my server:
https://solution-explorer.apps.cluster-uxddev-916c.uxddev-916c.example.opentlc.com/

**Comment by tiffanynolan** [comment]
Looking great! I noticed that the right side padding/gutter is missing from the data list. Can you add that back in? 
image

**Comment by tiffanynolan** [comment]
Also, there is some extra padding between the tab bar and the data list. Can this be removed?
Current: image
Desired: 
image

**Comment by mfrances17** [comment]
Fixed the issues @tiffanynolan found, put an image of the latest on my server. Please have a look at your earliest convenience:
https://solution-explorer.apps.cluster-uxddev-916c.uxddev-916c.example.opentlc.com/

Screen caps as well, for your convenience:

initial_page

settings_page

**Pull request merged**